packages/presentation/lib/screen/movie_details/widgets/details_background_image.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:domain/model/movie.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:presentation/config/theme/app_colors.dart';
+import 'package:presentation/utils/image_path.dart';
+
+class DetailsBackGroundImage extends StatelessWidget {
+  const DetailsBackGroundImage({
+    Key? key,
+    required this.movie,
+  }) : super(key: key);
+
+  final Movie? movie;
+
+  @override
```

Comment:
Decide on your entire project where you create variables - above or below the constructor?

Reply:
:)